.github/ISSUE_TEMPLATE/arf-eng-feature-issue.md

```diff
@@ -0,0 +1,49 @@
+---
+name: ARF Engineering Feature Issue
+about: Feature issue template for the ARF Engineering team
+title: ''
+labels: 'accredited-rep-facing', 'arf-eng'
+assignees: ''
+--
+
+### Description
+
+#### User Story or Problem Statement
+<!-- Outline the user story or the problem being addressed -->
+
+#### Implementation Details
+<!-- Describe how the feature or solution should be implemented -->
+
+#### Mockups/Designs
+<!-- Include any available mockups or design elements -->
+
+#### Prerequisites/Dependencies
+<!-- List any prerequisites or dependencies -->
+
+#### Blockers
+<!-- Identify any potential obstacles -->
+
+#### Specifications
+<!-- Detail the technical specifications required -->
+
+### Acceptance Criteria
+<!-- Specify criteria for ticket completion -->
+- [ ]
+
+#### Testing Criteria
+<!-- Outline the testing criteria (to be defined by the engineering team) -->
+
+#### Automated Testing
+<!-- Indicate the requirement for automated testing beyond unit tests -->
+
+#### Accessibility (A11y) Testing 
+<!-- Confirm that accessibility testing is completed -->
+
+#### Documentation
+<!-- Ensure relevant documentation is created or updated -->
+
+#### Monitoring
+<!-- Implement monitoring where appropriate (to be defined by the engineering team) -->
+
+#### PII/PHI Security
+<!-- Verify that personally identifiable information/protected health information is secure -->
```

Comment:
- As you know, I discussed with greater ARF team whether I should add required labels to important issue sections.
- It was decided that no section will have an explicit 'required' or 'optional' and therefore it will be decided by the person creating the issue which sections to include.
- If certain important sections are being continually omitted then we can add 'required' labels in a next iteration.